src/pat/autotoc/autotoc.js

```diff
@@ -121,15 +169,17 @@ export default Base.extend({
         });
 
         if (activeId) {
-            $("a#" + activeId).trigger("click", {
-                doScroll: true,
-                skipHash: true,
-            });
+            const activeTabButton = self.$toc.find("a#" + activeId)[0];
+            if (activeTabButton) {
+                const tab = Tab.getOrCreateInstance(activeTabButton);
+                tab.show();
+            }
         } else {
-            self.$toc.find("a").first().trigger("click", {
-                doScroll: false,
-                skipHash: true,
-            });
+            const firstTabButton = self.$toc.find("a").first()[0];
+            if (firstTabButton) {
+                const tab = Tab.getOrCreateInstance(firstTabButton);
+                tab.show();
+            }
         }
 
         // After DOM tree is built, initialize eventual validation
```

Comment:
you do not have to pass `self.$el` as parameter. Every pattern has `this.$el` (jQuery) and `this.el` (DOM native) available. So you can change line 190 to `const el = this.el`